src/steps/update-signatures/update-signature.ts

```diff
@@ -39,7 +40,14 @@ export function updateSignature(file: string, data: Data): string {
         return false;
       }
 
-      const ArgsNode = getBodyNode(path.node, 'Args');
+      const argsNode = getBodyNode(path.node, 'Args');
+
+      const isArgsKnown =
+        argsNode?.typeAnnotation.typeAnnotation.members.length > 0;
```

Comment:
**TODO**

This condition works if `Signature['Args']` is an object `{}` or the `'Args'` key is missing from `Signature`.

Check if there are edge cases for which the condition evaluates incorrectly or causes a runtime error.

Reply:
Completed in #36.

Reply:
Resolved in #36.